src/api/v1/paths/notes/{id}.js

```diff
@@ -67,6 +67,6 @@ const del = async (req, res) => {
 
 get.apiDoc = paths['/notes/{noteId}'].get;
 patch.apiDoc = paths['/notes/{noteId}'].patch;
-del.apiDoc = paths['/notes/{noteId}'].del;
+del.apiDoc = paths['/notes/{noteId}'].delete;
```

Comment:
This is what was causing the `req` parameter to be different for the delete endpoint and causing it to crash.